src/graphql/Characters.ts

```diff
@@ -0,0 +1,39 @@
+import { gql } from "@apollo/client";
+
+export const GET_CHARACTERS = gql`
```

Comment:
I would suggest you have each request in separate files

Reply:
Ok I got it.